protocols/virtualdefund/virtualdefund.go

```diff
@@ -304,6 +314,19 @@ func (o *Objective) Crank(secretKey *[]byte) (protocols.Objective, protocols.Sid
 		return &updated, sideEffects, WaitingForNothing, protocols.ErrNotApproved
 	}
 
+	if !updated.VoucherSent && updated.MyRole == payments.PAYER_INDEX {
```

Comment:
Comment this for clarity

Reply:
[ae60857](https://github.com/statechannels/go-nitro/pull/826/commits/ae608577cdd7f7c67cec911bf02b8dda7b6d5a09)